### PR TITLE
Gate drivers

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Adds two new gate driver symbols:

### L6491 

image

https://www.st.com/content/st_com/en/products/power-management/mosfet-and-igbt-gate-drivers/high-voltage-half-bridge-gate-drivers/l6491.html

### 2EDL23N06PJXUMA1

image

https://www.digikey.com/product-detail/en/infineon-technologies/2EDL23N06PJXUMA1/2EDL23N06PJXUMA1CT-ND/4571876

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

File: Driver_FET.dcm [hunk 1]
```diff
@@ -690,6 +702,12 @@ K Gate Driver
 F https://www.infineon.com/dgdl/Infineon-IRS2890DS-DS-v01_00-EN.pdf?fileId=5546d4625a888733015aad6fbc8a4bf4
 $ENDCMP
 #
+$CMP L6491
+D 600V Half Bridge Gate Driver with OCP and Integrated Bootstrap Diode, 4A drive current, DSO-14
+K Gate Driver MOSFET IGBT
+F https://www.st.com/content/ccc/resource/technical/document/datasheet/39/4a/2e/5f/bc/cf/42/20/DM00088170.pdf/files/DM00088170.pdf/jcr:content/translations/en.DM00088170.pdf
```
Comment (evanshultz) [inline Driver_FET.dcm]: Please change to `https://www.st.com/resource/en/datasheet/l6491.pdf`.

File: Driver_FET.dcm [hunk 2]
```diff
@@ -690,6 +702,12 @@ K Gate Driver
 F https://www.infineon.com/dgdl/Infineon-IRS2890DS-DS-v01_00-EN.pdf?fileId=5546d4625a888733015aad6fbc8a4bf4
 $ENDCMP
 #
+$CMP L6491
+D 600V Half Bridge Gate Driver with OCP and Integrated Bootstrap Diode, 4A drive current, DSO-14
```
Comment (evanshultz) [inline Driver_FET.dcm]: Change package to `SOIC-14`.

File: Driver_FET.lib [hunk 3]
```diff
@@ -1544,6 +1602,35 @@ X LO 9 300 -300 100 L 50 50 1 1 O
...
+$ENDFPLIST
+DRAW
+S -300 -400 300 400 0 1 10 f
+X ~LIN 1 -400 0 100 R 50 50 1 1 I
+X CP+ 10 400 -200 100 L 50 50 1 1 P
+X NC 11 -300 300 100 R 50 50 1 1 N N
+X OUT 12 400 100 100 L 50 50 1 1 P
+X HVG 13 400 200 100 L 50 50 1 1 O
+X BOOT 14 400 300 100 L 50 50 1 1 P
+X ~SD/~OD 2 -400 -100 100 R 50 50 1 1 C
```
Comment (evanshultz) [inline Driver_FET.lib]: The second `~` should be to the left of the `/`.

**Comment by evanshultz** [comment]
Please post a screenshot of `2ED21824S06J`.

File: Driver_FET.dcm [hunk 4]
```diff
@@ -1,5 +1,17 @@
 EESchema-DOCLIB  Version 2.0
 #
+$CMP 2ED21824S06J
+D 650V Half Bridge Gate Driver with Integrated Bootstrap Diode, 2.5A drive current, SO-14
+K Gate Driver MOSFET IGBT
+F https://www.infineon.com/dgdl/Infineon-2ED2182-4-S06F-J-DataSheet-v02_10-EN.pdf?fileId=5546d4626cb27db2016cb8d7368a29e3
+$ENDCMP
+#
+$CMP 2EDL23N06PJXUMA1
+D 600V Half Bridge Gate Driver with OCP and Integrated Bootstrap Diode, +1.8A/-2.5A drive current, DSO-14
```
Comment (evanshultz) [inline Driver_FET.dcm]: Package should be `SOIC-14`.

File: Driver_FET.dcm [hunk 5]
```diff
@@ -1,5 +1,17 @@
 EESchema-DOCLIB  Version 2.0
 #
+$CMP 2ED21824S06J
+D 650V Half Bridge Gate Driver with Integrated Bootstrap Diode, 2.5A drive current, SO-14
```
Comment (evanshultz) [inline Driver_FET.dcm]: Package should be `SOIC-14`.

**Comment by SchrodingersGat** [comment]
> 
> 
> Please post a screenshot of `2ED21824S06J`.

image

File: Driver_FET.dcm (hunk 1 above)
Comment (SchrodingersGat) [reply Driver_FET.dcm]: Done

File: Driver_FET.dcm (hunk 2 above)
Comment (SchrodingersGat) [reply Driver_FET.dcm]: Done

File: Driver_FET.lib (hunk 3 above)
Comment (SchrodingersGat) [reply Driver_FET.lib]: Done

File: Driver_FET.dcm (hunk 4 above)
Comment (SchrodingersGat) [reply Driver_FET.dcm]: Done

File: Driver_FET.dcm (hunk 5 above)
Comment (SchrodingersGat) [reply Driver_FET.dcm]: Done

**Comment by SchrodingersGat** [comment]
I think that is all outstanding issues addressed now :)

**Comment by evanshultz** [comment]
@SchrodingersGat 
Assuming my past self didn't miss anything, can you fix the branch conflict and I will merge? Apologies for missing this.

**Comment by evanshultz** [comment]
https://github.com/KiCad/kicad-symbols/pull/2877 added L6491, which was very similar but not quite the same as in this PR so it should be OK. I removed that symbol and fixed the branch conflict. Thanks!

**Pull request merged**